templates/features.html

```diff
@@ -116,16 +122,9 @@
   }
 });
 
-search.addEventListener('keyup', function(e) {
-  // Clear results.
-  if (!e.target.value && e.keyCode != 27) { // ESC
-    featureList.filter();
-    chromeMetadata.selected = null;
-  } else {
-    //if (e.keyCode == 13) {  // Enter
-      featureList.filter(e.target.value);
-    //}
-  }
+search.addEventListener('input', function(e) {
+  featureList.filter(e.target.value);
```

Comment:
does this change the UX of typing into the search box at all?  'input' definitely seems better than 'keyup' (eg. I think it'll also handle paste).   What's the implication of setting `chromeMetadata.selected=null` on every key press (where it looks like we used to only do that when the input field was empty)?